sparse/src/KokkosSparse_Utils.hpp

```diff
@@ -2339,8 +2339,6 @@ void kk_extract_diagonal_blocks_crsmatrix_sequential(
   using row_map_hostmirror_type = typename row_map_type::HostMirror;
   using entries_hostmirror_type = typename entries_type::HostMirror;
   using values_hostmirror_type  = typename values_type::HostMirror;
-  using int_view1d_type =
```

Comment:
Consider adding a doxygen-style comment to `kk_extract_diagonal_blocjks_crs_matrix_sequential` with a usage example as well as a unit-test.

Reply:
@e10harvey Thanks, I will .